kubetest2/pkg/app/testers/testers.go

```diff
@@ -59,3 +59,12 @@ func Register(name, usage string, tester types.NewTester) error {
 	}
 	return nil
 }
+
+// Names returns a slice of all registered tester names.
+func Names() []string {
```

Comment:
pretty sure this needs to hold the mutex. go will catch concurrent read/write at runtime and panic

Reply:
Donw.